### DISCUSSION TITLE
Rename CSS Vars to something more semantic

### DISCUSSION THREAD

**Discussion by ryansegus** [ismail9k/vue3-carousel#92] [Ideas]
Hello, nice job on the carousel, one little thing tough, I noticed that you have `--carousel-color-white` this can be easily change to something more meaningful, I propose `--carousel-color-arrows`.

In my case I wanted the arrows red with a background white so the naming looks weird:

```
--carousel-color-white: red; // This makes no sense
--carousel-color-arrows: red; // Better :)
```

I tried to push on your repository but I wasn't able, anyhow there just three changes:

examples.md
navigation.css
variables.css

Cheers!

**Comment by ismail9k** [comment]
Good idea actually, I will try to implement it.
